legacy/ExportPIM.js

```diff
@@ -153,7 +153,11 @@ function convertArrayOfObjectsToCSV(
       ) {
         recordAttributes.set(skey, recordAttributes.get(skey).Name);
       }
-      csvStringResult += cleanString(records[i].get(skey) || '');
+      if (typeof records[i].get(skey) === 'number') {
```

Comment:
do we want to put these values somewhere as a constant? same with other parts of the PR like 'Value__c', etc